src/Elastic.Markdown/Myst/InlineParsers/DiagnosticLinkInlineParser.cs

```diff
@@ -136,7 +136,7 @@ public override bool Match(InlineProcessor processor, ref StringSlice slice)
 			if (!string.IsNullOrEmpty(anchor))
 			{
 				if (markdown == null || (!markdown.TableOfContents.TryGetValue(anchor, out var heading)
-					&& !markdown.AdditionalLabels.Contains(anchor)))
+					&& !markdown.AdditionalLabels.Contains(anchor, StringComparer.OrdinalIgnoreCase)))
```

Comment:
@Mpdreamz What do you think about this?

What are additional labels actually?

Reply:
They are anchors discovered on the page. The property name is bad :)

Hashset supports a string comparer at creation time too. 

new HashSet<string>(StringComparer.OrdinalIgnoreCase);

Reply:
https://github.com/elastic/docs-builder/pull/341/commits/c7c0e6173252eb13b79ebca32231e6d585a96a30